src/ALCProxy.Communication/ALCClient.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using System.Reflection.Emit;
+using System.IO;
+
+namespace ALCProxy.Communication
+{
+    internal delegate object ServerCall(MethodInfo info, IList<object> args, IList<Type> types);
+    /// <summary>
+    /// This currently is designed to only work in-process
+    /// TODO: set up to allow for construction of out-of-proc proxies
```

Comment:
Move such TODO out of the xml comments (regular comment outside the `<summary>` is fine).